tools/start_all_tmux.sh

```diff
@@ -55,7 +59,18 @@ declare -a facerecognition=("FACE" "$(pwd)/../lucida/djinntonic/face")
 declare -a weather=("WE" "$(pwd)/../lucida/weather")
 declare -a musicservice=("MS" "$(pwd)/../lucida/musicservice")
 
-declare -a services=(
+if [ "$1" == "test" ]; then
+    declare -a services=(
+    questionanswering
+    imagematching
+    calendar
+    imageclassification
+    digitrecognition
+    facerecognition
+    weather
+    musicservice)
+else
```

Comment:
We want to avoid repeating the services here. That would mean that we would have to add a new microservice to each list. Let's figure out how to use only one list